uaclient/config.py

```diff
@@ -94,6 +104,27 @@ def log_level(self):
         except AttributeError:
             return getattr(logging, CONFIG_DEFAULTS["log_level"])
 
+    def add_notice(self, label: str, description: str):
+        notices = self.read_cache("notices") or []
```

Comment:
Maybe instead of a list of tuples, we could store the `notices` as a dict ? I don't think it will make the code faster, but the `remove_notice` method will be cleaner. But again, this is not a huge necessity at all.

Reply:
I think we need to pass on this until next round. Rick has a couple of ideas about where he wants these notices to go  and I think we may need to massage this data format a bit.